crates/tree-sitter-perl-c/c-src/scanner.c

```
@@ -0,0 +1,988 @@
+    return true;                 \
+  } while (0)
+
+#define MARK_END                        \
+  do {                                  \
+    lexer->mark_end(lexer);             \
+    DEBUG("marking end of token\n", 0); \
+  } while (0)
+
+static void skip_whitespace(TSLexer *lexer) {
+  while (1) {
+    int32_t c = lexer->lookahead;
```

Clang (14.0.6)

[warning] variable name 'c' is too short, expected at least 3 characters

(readability-identifier-length)